bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableClusterUtilities.java

```diff
@@ -97,17 +154,26 @@ public BigtableClusterUtilities(final BigtableOptions options)
    * @return the {@link Cluster#getServeNodes()} of the clusterId.
    */
   public int getClusterSize(String clusterId, String zoneId) {
+    Cluster cluster = getCluster(clusterId, zoneId);
     String message = String.format("Cluster %s/%s was not found.", clusterId, zoneId);
-    Cluster cluster = Preconditions.checkNotNull(getCluster(clusterId, zoneId), message);
+    Preconditions.checkNotNull(cluster, message);
     return cluster.getServeNodes();
   }
 
   /**
+   * Gets the serve node count of an instance with a single cluster.
+   * @return the {@link Cluster#getServeNodes()} of the clusterId.
+   */
+  public int getClusterSize() {
+    return getSingleCluster().getServeNodes();
+  }
+
+  /**
    * Gets a {@link ListClustersResponse} that contains all of the clusters for the
    * projectId/instanceId configuration.
-   * @return the current state of the instance
+   * @return the current state of the instance or the project if the instance is "-"
```

Comment:
"@return the current state of instance"? Please clarify, I thought this method returns a cluster list.